upi/aws/cloudformation/05_cluster_master_nodes.yaml

```diff
@@ -159,6 +165,11 @@ Resources:
     Type: AWS::EC2::Instance
     Properties:
       ImageId: !Ref RhcosAmi
+      BlockDeviceMappings:
+      - DeviceName: /dev/xvda
+        Ebs:
+          VolumeSize: "120"
+          VolumeType: "gp2"
```

Comment:
We bump the size in IPI to get more IOPS.  But it's not strictly necessary, and CI was looking reasonable without it.  Should we leave it off, per the minimal-necessary approach discussed in 9572a06d?  We aren't aiming for something that is all polished up out of the box.

Reply:
We want a usable cluster they can work with for some time and after deploying some of their own apps. Given how hard masters are to rotate, providing headroom here (or demonstrating what is needed in UPI to achieve it) should be helpful.